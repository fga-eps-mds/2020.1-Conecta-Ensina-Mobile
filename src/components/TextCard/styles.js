import styled from 'styled-components/native';

export const Container = styled.View`
  min-height: 40px;
  min-width: 150px;
  align-self: flex-end;
  background-color: #297fb8;
  align-items: center;
  justify-content: center;
  border-radius: 20px;
  margin-bottom: 10px;
`;

export const TextMessage = styled.Text`
  color: white;
`;
