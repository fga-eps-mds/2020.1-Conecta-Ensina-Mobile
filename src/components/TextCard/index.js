import React from 'react';
import {Container, TextMessage} from './styles';

export default function TextCard({data}) {
  return (
    <Container>
      <TextMessage>{data.text}</TextMessage>
    </Container>
  );
}
