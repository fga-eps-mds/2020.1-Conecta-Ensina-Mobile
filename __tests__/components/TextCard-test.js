import React from 'react';
import TextCard from '../../src/components/TextCard';
import {render} from '@testing-library/react-native';

describe('Test for TextCard component', () => {
  const data = {
    text: 'test',
  };
  test('should render component', () => {
    const tree = render(<TextCard data={data} />).toJSON;

    expect(tree).toMatchSnapshot();
  });
});
