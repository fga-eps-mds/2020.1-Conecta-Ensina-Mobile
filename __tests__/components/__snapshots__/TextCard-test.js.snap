// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for TextCard component should render component 1`] = `[Function]`;
